scania-digital-design-system/sdds
# feat(table): separation in sub-components

**Describe pull-request**  
Separating the data table in sub-components

**Solving issue**  
Prerequisite for expended table rows feature and easier maintenance

**How to test**  
1. Go to Storybook and open web component
2. Compare with prod Storybook to make sure non of functionality is damaged
